downstream/assemblies/platform/assembly-example-topologies.adoc

```diff
@@ -1,10 +1,10 @@
 
 ifdef::context[:parent-context: {context}]
 
-[id="assembly-standalone-controller-non-inst-database"]
+[id="assembly-example-topologies"]
```

Comment:
```suggestion
[id="design-patterns"]
```

Reply:
ID should match the title as closely as possible. Also avoid product names or putting attribute values (i.e. whatever `{AutomationMeshStart}` resolves to) in an ID because that can be derived from the URL context.